docs/adguard-vpn-for-ios/overview.md

```diff
@@ -43,6 +43,15 @@ You can add domains (e.g., `google.com`) or subdomains (e.g., `*.google.com`) of
 
 ![Exclusions *mobile_border](https://cdn.adguardvpn.com/content/kb/vpn/ios/2.2/exclusions-add.png)
 
+:::info Browsers in Selective mode
+
+Starting with the version 2.4, when switching to Selective mode, all browsers on the system are automatically added to the exceptions list for more convenience. This change applies under the following conditions:
```

Comment:
we don't have the "exceptions list". it's called Exclusions list: https://crowdin.com/editor/adguard-applications/all/en-ru?view=comfortable&filter=basic&value=0#q=exclusions%20list